integration_tests/seeds/netsuite_accounts_data.csv

```diff
@@ -1,11 +1,10 @@
-account_id,parent_id,name,type_name,account_number,general_rate_type,is_leftside,is_balancesheet
-530,,8769f2a4710dce527c26a6962560eb68,eb45bd2725cb8ac50d0795432a918f86,10000,a2770969c827f0f2910f6179418462df,No,No
-531,530,4f776429f32750022c01847c813b00cd,eb45bd2725cb8ac50d0795432a918f86,10001,a2770969c827f0f2910f6179418462df,No,No
-532,530,e17c5b5a8046fa938ae6c28848445e5a,eb45bd2725cb8ac50d0795432a918f86,10002,a2770969c827f0f2910f6179418462df,No,No
-533,530,8deba561c94866c69412c5ad9785955d,eb45bd2725cb8ac50d0795432a918f86,10003,a2770969c827f0f2910f6179418462df,Yes,Yes
-534,530,4d17a0007c857ebc8fa6be1e6d8e6eb2,eb45bd2725cb8ac50d0795432a918f86,10004,a2770969c827f0f2910f6179418462df,Yes,Yes
-535,530,8bc09cee29171553028ae27eebf6cdf5,eb45bd2725cb8ac50d0795432a918f86,10005,a2770969c827f0f2910f6179418462df,Yes,Yes
-1037,530,7790e9be35e2c705c2c62b04b503d962,eb45bd2725cb8ac50d0795432a918f86,10006,a2770969c827f0f2910f6179418462df,No,No
-1044,530,d739f290e21e40f880af009b06f115d2,eb45bd2725cb8ac50d0795432a918f86,10007,a2770969c827f0f2910f6179418462df,No,No
-1067,530,29c0ed5ce3ffa0115971fe05cd92e577,eb45bd2725cb8ac50d0795432a918f86,10008,a2770969c827f0f2910f6179418462df,Yes,Yes
-1050,530,f3537e12e19781fd44ca2b77be86afc8,eb45bd2725cb8ac50d0795432a918f86,10021,a2770969c827f0f2910f6179418462df,No,No
\ No newline at end of file
+530,,8769f2a4710dce527c26a6962560eb68,eb45bd2725cb8ac50d0795432a918f86,10000,a2770969c827f0f2910f6179418462df,f,f

```

Comment:
do we need to keep line 1 for the column names? it's odd that the `seed` and package ran correctly without it... 

though this is how the seeded table looks
<img width="1203" alt="image" src="https://github.com/fivetran/dbt_netsuite_source/assets/65564846/16d80dad-351b-46b3-960b-e19f5ee96558">

Reply:
Oops. This is fixed now.